storagemanager/storage/migrations/0003_auto_20200928_1620.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.1 on 2020-09-28 14:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
```

Comment:
This is a very minor, but since it won't be changed anywhere, I'd store tuples in tuple and not in list.